docs/performance.md

```diff
@@ -103,6 +103,8 @@ for f in 0..<2_000 {
 
 The above program contains the `TerribleArray` data structure which has _O(n)_ appends and not the amortised _O(1)_ that users are used to from `Array`.
 
+We will assume, that you have Linux's `perf` installed and configured, documentation on how to install `perf` can be found in [this guide](linux-perf.md).
```

Comment:
these changes seem unrelated?

Reply:
Not sure how those two commits got in there. I reverted both